scripts/corelli/calibration/database.py

```diff
@@ -123,3 +225,19 @@ def combine_spatial_banks(tables:WorkspaceGroup)->TableWorkspace:
             combined_table.addRow( [table_dict['Detector ID'][r], table_dict['Detector Position'][r] ])
```

Comment:
TableWorkspace `combined_table` should have a name, which usually is given as optional parameter `output_wokspace: str =  'some_name'` in the function's signature.